<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contador de Clicks</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(210, 199, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Resources/resource.qrc">:/logos/log_uat_nuevo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Resources/resource.qrc">:/logos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Resources/resource.qrc">:/logos/equipo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>395</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_1">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;Sitka Display&quot;;
color: rgb(13, 13, 85);
background-color: rgb(255, 153, 220);</string>
       </property>
       <property name="text">
        <string>Tata</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(13, 13, 85);
font: 75 11pt &quot;Sitka Display&quot;;
background-color: rgb(255, 153, 220);</string>
       </property>
       <property name="text">
        <string>Cooky</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(13, 13, 85);
font: 75 11pt &quot;Sitka Display&quot;;
background-color: rgb(255, 153, 220);</string>
       </property>
       <property name="text">
        <string>Rj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(13, 13, 85);
background-color: rgb(255, 153, 220);
font: 75 11pt &quot;Sitka Display&quot;;</string>
       </property>
       <property name="text">
        <string>Chimmy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_imagen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>395</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Resources/resource.qrc">:/Ejercicios/tata.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_instruccion">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(13, 13, 85);
font: 75 11pt &quot;Sitka Display&quot;;
background-color: rgb(217, 241, 3);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>     Memoriza el patrón y selecciona la opción correcta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivos/Resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
